src/test/java/no/unit/nva/model/contexttypes/BookTest.java

```diff
@@ -0,0 +1,64 @@
+package no.unit.nva.model.contexttypes;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import no.unit.nva.model.Level;
+import nva.commons.utils.IoUtils;
+import nva.commons.utils.JsonUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class BookTest {
+
+    public static final ObjectMapper objectMapper = JsonUtils.objectMapper;
+    public static final String BOOK_JSON = "book.json.template";
+    public static final int FIRST_ELEMENT = 0;
+
+    @DisplayName("Book can deserialize a book")
+    @Test
+    void objectMapperReturnsBookWhenInputIsValidJson() throws JsonProcessingException {
+        String expectedType = "Book";
+        String expectedTitle = "The history of JSON in this project";
+        Level expectedLevel = Level.LEVEL_2;
+        String expectedIsbn = "123 123123 1112";
+        String json = generateJsonFromTemplate(expectedType, expectedTitle, expectedLevel,
+                true, true, expectedIsbn);
+        Book book = objectMapper.readValue(json, Book.class);
```

Comment:
Suggestion: Create an expected book instance and do assertEquals(expectedBook, actualBook)?
If you also refactor the generation of an expected book to a method, you get an (almost) free test on equals as well :)